src/main/spark-1.x/org/apache/spark/graphx/lib/GraphXHelpers.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.graphx.lib
+
+import scala.reflect.ClassTag
+
+import breeze.linalg.{Vector => BV}
+
+import org.apache.spark.graphx._
+import org.apache.spark.mllib.linalg.Vector
+
+object GraphXHelpers {
+  def runParallelPersonalizedPageRank[VD: ClassTag, ED: ClassTag] (
+      graph: Graph[VD, ED],
+      numIter: Int, 
+      resetProb: Double,
+      sources: Array[VertexId]): Graph[Vector, Double] = {
```

Comment:
I'd search & replace sources with sourceIds everywhere.  Consistency will help make refactoring easier in the future